src/platforms/javascript/common/sourcemaps/troubleshooting_js.mdx

```diff
@@ -31,6 +31,18 @@ Double-check that your deployed, final JavaScript files have `sourceMappingURL`
 
 Alternately, instead of `sourceMappingURL`, you can set a `SourceMap` HTTP header on your minified file. If this header is present, Sentry will use it to discover the location of your source map.
 
+<PlatformSection notSupported={["javascript.electron"]}>
+
+## Verify artifacts distribution match value configured in your SDK
+
+Whenever you are using distribution identifier (`dist` configuration option in the SDK), it has to be matched during the source maps upload as well.
```

Comment:
```suggestion
Whenever you are using a distribution identifier (`dist` configuration option in the SDK), the same value has to be used during the source maps upload as well.
```

Reply:
Building on Abhi's suggestion (so this includes his and mine).

```suggestion
Whenever you are using a distribution identifier (the `dist` configuration option in the SDK), the same value has to be used during source map upload. Conversely, if your source maps are getting uploaded with a `dist` value, the same value must be set in your SDK.
```